src/lib/inputs/InputRadio/InputRadio.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  Radio as MuiRadio,
+  RadioProps,
+  FormControlLabel as MuiFormControlLabel,
+  FormControlLabelProps
+} from '@material-ui/core';
+import classNames from 'classnames';
+import { FormikProps, FormikValues } from 'formik';
+import './input-radio.scss';
+
+export type InputRadioProps = {
+  label: string;
+  labelPlacement?: FormControlLabelProps['labelPlacement'];
+  formikProps?: FormikProps<FormikValues>;
+} & RadioProps;
+
+export const InputRadio: React.FC<InputRadioProps> = ({
+  className,
+  color = 'primary',
+  formikProps,
+  label,
+  labelPlacement,
+  onChange,
+  value,
+  ...props
+}) => {
+  const fieldProps = formikProps?.getFieldProps(props.name);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (onChange) onChange;
+    if (fieldProps?.onChange) fieldProps.onChange(event);
+  };
+
+  return (
+    <MuiFormControlLabel
+      className={classNames('lc-input lc-input-radio', className)}
+      label={label}
+      labelPlacement={labelPlacement}
+      control={
+        <MuiRadio
+          {...props}
+          color={color}
+          checked={fieldProps ? fieldProps?.value === value : props.checked}
```

Comment:
Since we are using optional chaining, it seems like we could/should do:

```
checked={fieldProps?.value === value || props.checked}
```

Reply:
Or maybe:

```
checked={(fieldProps && fieldProps?.value === value) || props.checked}
```

since we might want `props.checked` to still apply if both `fieldProps?.value` and `value` happen to be `undefined`.

Reply:
Note we don't need the optional chaining, if we are already checking for `fieldProps` before.